server/core/src/https/javascript.rs

```diff
@@ -34,24 +34,33 @@ pub fn generate_integrity_hash(filename: String) -> Result<String, String> {
 pub struct JavaScriptFile {
     // Relative to the pkg/ dir
     pub filepath: &'static str,
+    // Dynamic
+    pub dynamic: bool,
     // SHA384 hash of the file
     pub hash: String,
     // if it's a module add the "type"
     pub filetype: Option<String>,
 }
 
 impl JavaScriptFile {
-    /// returns a `<script>` HTML tag
+    /// returns a `<script>` or `<meta>` HTML tag
     pub fn as_tag(&self) -> String {
         let filetype = match &self.filetype {
             Some(val) => {
                 format!(" type=\"{}\"", val.as_str())
             }
             _ => String::from(""),
         };
-        format!(
-            r#"<script async src="/pkg/{}" integrity="sha384-{}"{}></script>"#,
-            self.filepath, &self.hash, &filetype,
-        )
+        if self.dynamic {
```

Comment:
I'm a web-boomer, what's the different here with these two types of load?

Reply:
This doesn't actually load the script, the meta tag is used as a placeholder so it can be replaced later with the script tag using `init_graphviz` from `server/web_ui/shared/static/shared.js`. It's a bit of a hack but it works.